trackr-app/app/src/main/java/com/trackr/trackr_app/usecases/UserManager.kt

```diff
@@ -0,0 +1,34 @@
+package com.trackr.trackr_app.usecases
+
+import com.trackr.trackr_app.entities.User
+import java.util.UUID
+
+/**
+ * UserManager manages users
+ */
+class UserManager {
+    /**
+     * Create a new user
+     * @param id - unique user identifier
+     * @param username - whatever the user wants to call themselves
+     */
+    fun createUser(
+            id: UUID,
```

Comment:
I think maybe the user manager should be generating the id rather than passing it in.

Reply:
Ok. At the time i figured generating UUIDs might have to be its own method but we can do so in each method without issue.